internal/services/servicefabric/service_fabric_cluster_resource_test.go

```diff
@@ -1371,12 +1371,18 @@ data "azurerm_client_config" "current" {
 }
 
 resource "azuread_application" "cluster_explorer" {
-  name                       = "${azurerm_resource_group.test.name}-explorer-AAD"
-  homepage                   = "https://example:19080/Explorer/index.html"
-  identifier_uris            = ["https://example%d:19080/Explorer/index.html"]
-  reply_urls                 = ["https://example:19080/Explorer/index.html"]
-  available_to_other_tenants = false
-  oauth2_allow_implicit_flow = true
+  display_name    = "${azurerm_resource_group.test.name}-explorer-AAD"
+  identifier_uris = ["https://example%d:19080/Explorer/index.html"]
```

Comment:
It looks like we're bumping up against the new validation for identifier URIs - they now have to use a verified domain of the tenant. I think we can use the `azuread_domains` data source to configure this.

```hcl
data "azuread_domains" "test" {}
```

and then:
```suggestion
  identifier_uris = ["https://${data.azuread_domains.test.domains[0].domain_name}:19080/Explorer/index.html"]
```

Reply:
Smart, thank you! 👍🏽